### DISCUSSION TITLE
"Delete left file" with keyboard?

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1606] [General]
I compare a folder and a ZIP, and when I find two identical files I want to delete the file in the folder. With menu it works as shown on the image: "Delete - only left"
![grafik](https://user-images.githubusercontent.com/13277944/207382449-85bfdc37-2e32-4cd0-94c6-4d32753031c3.png)
Can this be done also with Hotkey or single keyboard (like "Del")?

**Comment by sdottaka** [comment]
Unfortunately there is no shortcut key for that menu item.
To select that menu on your keyboard, press Shift+F10→D→L.

**Comment by meteorquake** [comment]
I think there's a little merit in the suggestion, as when comparing things you are often adjusting one folder to another with the deletions on one side (but which...) although sometimes on multiple sides.
However normally you would just ctrl-click to highlight all the files you want to delete then use the context menu, which isn't too hard unless you are checking them one-by-one and deleting.
An alternative suggestion is that you could have some extra columns people can show that can have action buttons on. For example, if you show a column called, say, Del Left, it would provide a button for each line that you could just click on to do the action. Same for all the other actions.
Or, better, you could have a couple of general purpose action columns which you can show/hide and assign to different actions, thereby creating action buttons you are interested in shown against every file.
David

**Reply by Pierre-de-la-Verre** [reply]
> ... An alternative suggestion is that you could have some extra columns people can show that can have action buttons on. ...

Yes, that would be fine. For example, TotalCommander TC offers a great menu in the result of the "synchronization" feature.
